src/snapred/backend/data/LocalDataService.py

```diff
@@ -448,6 +448,16 @@ def readCalibrationRecord(self, runId: str, version: str = None):
             record = parse_file_as(CalibrationRecord, latestFile)
         return record
 
+    def readCalibrationRecords(self, runId: str) -> CalibrationRecordList:
```

Comment:
My understanding from talking to Malcolm is that he expects to select a subset from an available list of strings, e.g.
Run 1234 12/06/2023
Run 56789 12/07/2023
etc.

So that means the backend should be able to:
(1) produce such a list given a single run number (without loading/creating any workspaces)
(2) load/generate all relevant workspaces on user's request, where the request might contain more than one run id